.build/colors.mk

```diff
@@ -1,18 +0,0 @@
-# Colorization rules
```

Comment:
RIP my proudest Makefile moment :(. 

I know it's not used post di refactor, but this file makes the make output kind of awesome. You're welcome to drop it but it will make a return 🤘